workspaces/npm/plugins/npm/src/components/NpmInfoCard.tsx

```diff
@@ -0,0 +1,208 @@
+ * the latest version if available.
+ *
+ * @public
+ */
+export function NpmInfoCard() {
+  const { entity } = useEntity();
+
+  const packageName = entity.metadata.annotations?.[NPM_PACKAGE_ANNOTATION];
+
+  const packageInfo = useAsync(
+    () => API.fetchNpmPackage(packageName),
+    [packageName],
+  );
+
+  if (!packageName) {
+    return (
+      <MissingAnnotationEmptyState
+        annotation={NPM_PACKAGE_ANNOTATION}
+        readMoreUrl="https://backstage.io/docs/features/software-catalog/descriptor-format"
+      />
+    );
+  }
+
+  const latestTag =
+    entity.metadata.annotations?.[NPM_STABLE_TAG_ANNOTATION] ?? 'latest';
+  const latestVersion = packageInfo.value?.['dist-tags']?.[latestTag];
+  const latestPublishedAt = latestVersion
+    ? packageInfo.value?.time?.[latestVersion]
+    : undefined;
+
+  const npmLink = `https://www.npmjs.com/package/${packageName}`;
+
+  let repositoryLink: string | undefined;
+  if (packageInfo.value?.repository?.url) {
+    let url = packageInfo.value.repository.url;
+    if (url.startsWith('git+https://')) {
+      url = url.slice('git+'.length);
+    }
+    if (url.endsWith('.git')) {
+      url = url.slice(0, -'.git'.length);
+    }
+    if (url.startsWith('https://')) {
+      if (
+        url.startsWith('https://github.com/') &&
+        packageInfo.value.repository.directory
+      ) {
+        repositoryLink = `${url}/tree/main/${packageInfo.value.repository.directory}`;
+      } else {
+        repositoryLink = url;
+      }
+    }
+  }
+
+  const bugsLink = packageInfo.value?.bugs?.url;
+
+  const homepageLink = packageInfo.value?.homepage;
+
+  return (
+    <Card>
+      <CardHeader title={`Npm package ${packageName}`} />
+      <Divider />
+      <CardContent>
+        <Grid container>
+          {latestVersion ? (
+            <Item label="Latest version" value={latestVersion} md={4} />
+          ) : null}
+
+          {latestPublishedAt ? (
+            <Item
+              label="Published at"
+              value={
+                <time dateTime={latestPublishedAt} title={latestPublishedAt}>
+                  {DateTime.fromISO(latestPublishedAt).toRelative()}
+                </time>
+              }
+              md={4}
+            />
+          ) : null}
+
+          {packageInfo.value?.license ? (
+            <Item label="License" value={packageInfo.value.license} md={4} />
+          ) : null}
+
+          {/* Markdown? */}
+          {packageInfo.value?.description ? (
+            <Item label="Description" value={packageInfo.value.description} />
+          ) : null}
+
+          {/* Markdown? */}
+          {packageInfo.value?.keywords?.length ? (
+            <Item
+              label="Keywords"
+              value={packageInfo.value.keywords.join(', ')}
+            />
+          ) : null}
+
+          {npmLink ? (
+            <Item
+              label="Npm repository"
+              value={<Link href={npmLink}>{npmLink}</Link>}
```

Comment:
nit: Can we open all these links in the npm info card on a new tab so that we dont move away from the application?
```suggestion
              value={<Link href={npmLink} target="_blank">{npmLink}</Link>}
```